docker/Dockerfile

```diff
@@ -80,6 +85,8 @@ EXPOSE 2181 2888 3888 8080
 ENV PATH=$PATH:/$DISTRO_NAME/bin \
     ZOOCFGDIR=$ZOO_CONF_DIR
 
-COPY docker-entrypoint.sh /
-ENTRYPOINT ["/docker-entrypoint.sh"]
+COPY zookeeper.py .
+COPY docker-entrypoint.sh .
+
+ENTRYPOINT [ "./docker-entrypoint.sh" ]
 CMD ["zkServer.sh", "start-foreground"]
```

Comment:
I think, this is what we need for the `helix-ui-server` as well right.

Reply:
Most likely, I have to look at the it again.